Source/DeepRim/GenStep_RocksFromGrid_Deep.cs

```diff
@@ -57,7 +57,7 @@ public override void Generate(Map map, GenStepParams parms)
         }
 
         var genStep_ScatterLumpsMineable = new GenStep_ScatterLumpsMineable();
-        var num = 16f;
+        var num = DeepRimMod.instance.DeepRimSettings?.OreDensity != null ? (float)DeepRimMod.instance.DeepRimSettings?.OreDensity : 16f;
```

Comment:
I don't know if the line needs to be this long but I was getting hard exceptions when generating a new layer if I didn't check for null 🤷.
Is because OreDensity doesn't have a getter so it might actually be null if the user hasn't gone into that menu.... idk.